docs/wallet-connect-testing.md

```diff
@@ -0,0 +1,68 @@
+# WalletConnect Testing
+
+This document describes how to test our WalletConnect integration.
+
+As a wallet you can use the Use [Metamask][] or [Rainbow][] mobile apps or the
+WalletConnect [browser test wallet][test-wallet].
+
+## Connect a wallet
+
+1. Go to wallet screen in the app.
+2. Ensure you are disconnected. If that is not the case, disconnect and refresh
+   the app.
+3. Click “Connect your wallet”.
+4. Scan the QR code or paste the link into the wallet.
+5. Approve the connection in the wallet.
+6. The Wallet screen should show the correct address from the wallet
+
+## Disconnect from the app
+
+1. Ensure that you are connected with a wallet
+2. Click on the cross in the wallet screen to disconnect.
+3. You should see the screen that asks you to connect to a wallet.
+4. Reconnect with the wallet again.
+
+## Disconnect from the wallet
+
+This requires the browser test wallet. Disconnecting from the wallet is not
+possible in other wallet.
+
+1. Ensure that you are connected with the browser test wallet.
+2. Click on “disconnect” in the browser test wallet.
+3. In the app, you should see the screen that asks you to connect to a wallet.
+4. Reconnect with the wallet again.
+
+## Close connection modal
+
+1. Ensure you are disconnected.
+2. Click on “Connect your wallet”.
+3. Close the modal by clicking outside of it.
+4. Click on “Connect your wallet” and reconnect with a wallet again.
+
+## Switch network
```

Comment:
Do we need a similar test for switching the network in upstream settings and seeing what happens?

Reply:
The code that handles upstream network changes does not involve wallet connect. This is something we can test automatically later.

Reply:
> are they valuable in the first place?

It’s indeed dubious whether this is valuable. We want to get rid of this anyways https://github.com/radicle-dev/radicle-upstream/issues/2002

> Maybe mocking WC could allow that thinking

Yes, that is what I’m working on.